packages/react-menu/src/components/MenuList/MenuList.test.tsx

```diff
@@ -3,11 +3,18 @@ import { MenuList } from './MenuList';
 import * as renderer from 'react-test-renderer';
 import { ReactWrapper } from 'enzyme';
 import { isConformant } from '../../common/isConformant';
+import { MenuListProvider } from '../../menuListContext';
 
 describe('MenuList', () => {
   isConformant({
+    asPropHandlesRef: true,
     Component: MenuList,
     displayName: 'MenuList',
+    helperComponents: [MenuListProvider],
+    requiredProps: {
+      onCheckedValueChange: jest.fn(),
+      checkedValues: {},
+    },
```

Comment:
I don't think that this will be required if we will have a proper default values on context. This should also simplify testing for consumers...

Reply:
This has nothing to do with context values since, context isn't even used in this component, but they are not required so I will remove